plugin/src/test/java/io/jenkins/plugins/redis/RedisFingerprintCleanupTest.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2020, Jenkins project contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package io.jenkins.plugins.redis;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+import hudson.Util;
+import hudson.model.Fingerprint;
+import hudson.model.TaskListener;
+import jenkins.fingerprints.FingerprintCleanupThread;
+import org.jenkinsci.main.modules.instance_identity.InstanceIdentity;
+import org.junit.After;
```

Comment:
Since this is a new plugin, I recommend removing this to junit5 (aka jupiter). We can leverage @ParameterizedTest to make the tests cleaners. I also suggest doing that in a separate PR

Reply:
Noted, I will introduce a separate PR for that :)

Reply:
@afalko I was looking into this, it seems that JUnit 5 is not supported by Jenkins Test Harness atm (see https://issues.jenkins-ci.org/browse/JENKINS-48466). I will look around to see if there is some work around. 

https://junit.org/junit5/docs/current/user-guide/#migrating-from-junit4-rule-support exists but it says it is experimental

Reply:
Ahh, bummer. Thanks for checking! We should defer junit5 work till last phase if there is any time remaining. I think JENKINS-48466 will be high impact for community.